packages/core/cache/nuxt/index.js

```diff
@@ -1,93 +1,117 @@
 /* eslint-disable */
-import path from 'path';
-import { createMiddleware } from '@vue-storefront/core/server';
+import { isAbsolute, resolve } from 'path';
 import { Logger } from '@vue-storefront/core';
 
-function rendererFactory (renderFn) {
+/**
+ * Adds endpoint to invalidate cache
+ */
+function createInvalidationEndpoint (driver, options) {
+  if (!options || !options.endpoint || !options.key) {
+    return;
+  }
+
+  this.addServerMiddleware({
+    path: options.endpoint,
+    handler: async (request, response) => {
+      try {
+        // Remove leading slash and get URL params
+        const params = new URLSearchParams(request.url.replace(/^\//, ''));
```

Comment:
remove lines from 18 to 24 - you can't assume that someone as an invalidator will send you tags in the URL - imageina a webhooks from certain commerce platform or others - based on their payload you need to build tags